chain/client/src/client_actor.rs

```diff
@@ -1063,6 +1065,19 @@ impl ClientActor {
 
         let timer = metrics::CHECK_TRIGGERS_TIME.start_timer();
         if self.sync_started {
+            self.sync_timer_next_attempt = self.run_timer(
+                self.sync_wait_period(),
+                self.sync_timer_next_attempt,
+                ctx,
+                |act, _| act.run_sync_step(),
+                "sync",
+            );
+
+            delay = core::cmp::min(
+                delay,
+                self.sync_timer_next_attempt.signed_duration_since(now).to_std().unwrap_or(delay),
+            );
+
```

Comment:
doomslug_timer_next_attempt shouldn't be inside in the if

Reply:
it was in the 'if' before..

Reply:
oh... I see.